Libraries/Utilities/AcessibilityMapping.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+// Map role values to AccessibilityRole values
+export const roleToAccessibilityRoleMapping = {
```

Comment:
This object `roleToAccessibilityRoleMapping` needs to in JS VM heap all time after loading this JS. Consider  making this is getter function so this object can be deallocated after the call, or better use a switch statement like this. 
https://github.com/facebook/react-native/blob/84225573c17b17844b3b3e2a438b3178f7d35b05/packages/normalize-color/index.js#L303

cc @javache

Reply:
Sure, I've just refactor it to be a switch statement